linux/tools.Dockerfile

```diff
@@ -5,58 +5,40 @@
 
 # To build yourself locally, override this location with a local image tag. See README.md for more detail
 
-ARG IMAGE_LOCATION=cdpxb787066ec88f4e20ae65e42a858c42ca00.azurecr.io/official/azure/cloudshell:1.0.20220715.1.base.master.86bf8742
+ARG IMAGE_LOCATION=cdpxb787066ec88f4e20ae65e42a858c42ca00.azurecr.io/official/azure/cloudshell:1.0.20220720.1.base.mariner_full.d6d09980
```

Comment:
once the PR is merged will we do a build of the base and update this?

Reply:
+1

Reply:
@robins1212 Yes, we will do a build of the base and connect to the new ACR link.